pkg/config/config_test.go

```diff
@@ -913,4 +913,19 @@ env=["foo=bar"]`
 		err := ValidateImageVolumeMode("bogus")
 		gomega.Expect(err).To(gomega.HaveOccurred())
 	})
+
+	It("CONTAINERS_CONF_OVERRIDE", func() {
+		os.Setenv("CONTAINERS_CONF_OVERRIDE", "testdata/containers_override.conf")
+		defer os.Unsetenv("CONTAINERS_CONF_OVERRIDE")
+		config, err := NewConfig("")
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		gomega.Expect(config.Containers.ApparmorProfile).To(gomega.Equal("overridden-default"))
```

Comment:
Let's pretend that the `OVERRIDE` code breaks, and becomes equivalent to `$CONTAINERS_CONF` (i.e. ignoring existing conf). This test would happily pass.

Suggestion: include a test for a known-good setting, something that `containers_override.conf` does not explicitly set. (I don't know what environment this test runs in. It might be necessary to add this new check after line 929 below, something that is set in `containers_default.conf`). Left as exercise for the reader.